tests/runner_templates/checkout-test-plots.include.yml

```diff
@@ -7,9 +7,16 @@
           ${{ github.workspace }}/.chia/test-plots
         key: 0.29.0
 
-    - name: Checkout test blocks and plots
-      if: steps.test-blocks-plots.outputs.cache-hit != 'true'
+    - name: Checkout test blocks and plots (Linux, macOS)
+      if: steps.test-blocks-plots.outputs.cache-hit != 'true' && (startswith(matrix.os, 'ubuntu') || startswith(matrix.os, 'macos'))
```

Comment:
Wouldn't it be possible to merge those into one step and have different run sections depending on the OS? Doesn't matter much just thought it might be cleaner in the action overview page :)

Reply:
Got it. I just thought there might be some way to do something like 

```
- name: Checkout test blocks and plots
   if: macos/linux
   run: ....
   if windows:
   run: ....
```

But i don't know much about github actions :)